writing-apps/our-first-app/the-build-system.md

```diff
@@ -71,16 +71,21 @@ If you were about to add the "build" folder to your git repository and push it,
 
 ## Uninstalling the application
 
-To uninstall your application, we need to remove the several files we've created outside of our project folder:
+To uninstall your application, change into the build directory and we will use `ninja` once again to uninstall:
 
-* Binary file located at `/usr/bin/com.github.yourusername.hello-again`
-* Desktop file in `/usr/share/applications/com.github.yourusername.hello-again.desktop`
-* Application Data file in `/usr/share/metainfo/com.github.yourusername.hello-again.appdata.xml`
+```bash
+cd build
+sudo ninja uninstall
```

Comment:
I'm not sure, but I _think_ you can just `ninja uninstall` here as well, and it will elevate with polkit? I'd need to double-check, though.

Reply:
No, `ninja uninstall` do not use polkit, so sudo is needed.